nltk/metrics/association.py

```diff
@@ -61,16 +64,14 @@ class NgramAssocMeasures(object):
     _n = 0
 
     @staticmethod
+    @abstractmethod
     def _contingency(*marginals):
         """Calculates values of a contingency table from marginal values."""
-        raise NotImplementedError("The contingency table is not available"
-                                    "in the general ngram case")
```

Comment:
For these cases I think the error message is important. Is there a way to use `@abstractmethod` and still raise the error message?